scipy/stats/_continuous_distns.py

```diff
@@ -3025,8 +3058,230 @@ def alphanot1func(alpha, beta, TH, aTH, bTH, cosTH, tanTH, W):
+        fft_grid_spacing = 0.01 if not hasattr(self, 'pdf_fft_grid_spacing') else self.pdf_fft_grid_spacing
+        fft_n_points_two_power = None if not hasattr(self, 'pdf_fft_n_points_two_power') else int(self.pdf_fft_n_points_two_power)
+        
+        # group data in unique arrays of alpha, beta pairs
+        uniq_param_pairs = np.vstack({tuple(row) for row in data_in[:,1:]})
+        for pair in uniq_param_pairs:
+            data_mask = np.all(data_in[:,1:] == pair, axis=-1)
+            data_subset = data_in[data_mask]
+            if len(data_subset) < fft_min_points_threshold:
+                data_out[data_mask] = [pdf_single_value_with_quad(lambda t: levy_stable_gen._cf(t, _alpha, _beta), _x) 
+                            for _x, _alpha, _beta in data_subset]
+            else:
+                _alpha, _beta = pair
+                _x = data_subset[:,(0,)]
+                
+                # need enough points to "cover" _x for interpolation
+                h = fft_grid_spacing
+                q = np.ceil(np.log(2*np.max(np.abs(_x))/h)/np.log(2)) + 2 if fft_n_points_two_power is None else fft_n_points_two_power
+                
+                density_x, density = levy_stable_gen._pdf_from_cf_with_fft(lambda t: levy_stable_gen._cf(t, _alpha, _beta), h=h, q=q)
+                f = interpolate.interp1d(density_x, density)
+                data_out[data_mask] = np.real(f(_x))
+                
+        return data_out.T[0]
+
+    def _cdf(self, x, alpha, beta):
+
+        x = np.asarray(x).reshape(1, -1)[0,:]
+        
+        if np.shape(x) != np.shape(alpha):
+            alpha = np.full(np.shape(x), alpha)
+            
+        if np.shape(x) != np.shape(beta):
+            beta = np.full(np.shape(x), beta)
+
+        data_in = np.dstack((x, alpha, beta))[0]
+        data_out = np.empty(shape=(len(data_in),1))
+        
+        fft_grid_spacing = 0.01 if not hasattr(self, 'pdf_fft_grid_spacing') else self.pdf_fft_grid_spacing
+        fft_n_points_two_power = None if not hasattr(self, 'pdf_fft_n_points_two_power') else int(self.pdf_fft_n_points_two_power)
+        
+        # group data in unique arrays of alpha, beta pairs
+        uniq_param_pairs = np.vstack({tuple(row) for row in data_in[:,1:]})
+        for pair in uniq_param_pairs:
+            data_mask = np.all(data_in[:,1:] == pair, axis=-1)
+            data_subset = data_in[data_mask]
+
+            _alpha, _beta = pair
+            _x = data_subset[:,(0,)]
+            
+            # need enough points to "cover" _x for interpolation
+            h = fft_grid_spacing
+            q = 16 if fft_n_points_two_power is None else fft_n_points_two_power
+            
+            density_x, density = levy_stable_gen._pdf_from_cf_with_fft(lambda t: levy_stable_gen._cf(t, _alpha, _beta), h=h, q=q)
+            f = interpolate.InterpolatedUnivariateSpline(density_x, density)
```

Comment:
This needs to match the spline as obtained by interp1d. Would the two you suggest do that?